MIGRATION.md

```diff
@@ -167,6 +168,55 @@ Integrations that are now exported from `@sentry/node` and `@sentry/browser` (or
 
 The `Transaction` integration has been removed from `@sentry/integrations`. There is no replacement API.
 
+#### @sentry/serverless
+
+`@sentry/serverless` has been removed and will no longer be published. The serverless package has been split into two
+different packages, `@sentry/aws-serverless` and `@sentry/google-cloud`. These new packages have smaller bundle size
+than `@sentry/serverless`, which should improve your serverless cold-start times.
+
+`@sentry/aws-serverless` and `@sentry/google-cloud` has also been changed to only emit CJS builds. The ESM build for the
+`@sentry/serverless` package was always broken and we decided to remove it entirely. ESM support will be re-added at a
+later date.
+
+In `@sentry/serverless` you had to use a namespace import to initialize the SDK. This has been removed so that you can
+directly import from the SDK instead.
+
+```js
+// v7
+const Sentry = require('@sentry/serverless');
+
+Sentry.AWSLambda.init({
+  dsn: '__DSN__',
+  tracesSampleRate: 1.0,
+});
+
+// v8
+const Sentry = require('@sentry/aws-serverless');
+
+Sentry.init({
+  dsn: '__DSN__',
+  tracesSampleRate: 1.0,
+});
+```
+
+```js
+// v7
+const Sentry = require('@sentry/serverless');
+
+Sentry.GCPFunction.init({
+  dsn: '__DSN__',
+  tracesSampleRate: 1.0,
+});
+
+// v8
+const Sentry = require('@sentry/google-cloud');
```

Comment:
Small thing that comes to my mind, naming feels a bit inconsistent, with `@sentry/google-cloud` and `@sentry/aws-serverless` 🤔 feels like either both should have a `serverless` suffix (or prefix?), or not...?

Reply:
Good point.

We should rename `@sentry/google-cloud` to be `@sentry/google-cloud-serverless`, I can do that in a follow up PR. Adding task for it to issue.

The reason we want `-serverless` suffix is because we don't want to confuse people into thinking these packages instrument aws/gcp node SDK generally.

Reply:
yeah I think that's fair, let's suffix them both then, sounds good to me!